.editorconfig

```diff
@@ -0,0 +1,44 @@
+root = true
+
+[*]
+charset = utf-8
+end_of_line = lf
+indent_size = 2
+indent_style = space
+insert_final_newline = true
+trim_trailing_whitespace = true
+
+[{*.enc,*.go,*.gpg,*.md,*.svg}]
+trim_trailing_whitespace = false
```

Comment:
Why are we not trimming trailing whitespace from `.go` files?

Reply:
`gofmt` takes care of go files, this does not work well with multiline strings

Reply:
This was carried over from the `lorawan-stack` repo, should it be deleted? The comments are not clear to me.